README.md

```diff
@@ -60,6 +60,66 @@ jupyter lab
 
 3. Run all cells (Kernel > Restart & Run All) to reproduce the analysis and generate the visualizations.
 
+## Dependencies
+
+[Docker](https://www.docker.com/) is a container solution used to manage the software dependencies for this project. The Docker image used for this project is based on the condaforge/miniforge3:latest image. Additional dependencies are specified in the [Dockerfile](Dockerfile)
+
+### Installing Docker
+
+If you don't have Docker installed, download and install it from:
+- **Windows/Mac**: [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- **Linux**: Follow the [official installation guide](https://docs.docker.com/engine/install/)
+
+To verify Docker is installed correctly, run:
+```bash
+docker --version
+```
+
+### Other Requirements
+- Wine dataset (if not included in the container)
+
+## Quick Start
+
+## Running the Analysis
+
+### Step 1: Clean the Project
+
+Navigate to the root of this project on your computer using the command line and enter the following command to reset the project to a clean state (i.e., remove all files generated by previous runs of the analysis):
+
+```bash
+docker-compose run --rm analysis-env make clean
+```
+
+### Step 2: Run the Full Analysis
```

Comment:
This is correct but I think we instruct the user to run the image they just pulled. After pulling the image, when user runs `docker pull <name-of-our-image>:<tags>', the image will run and open the notebook by default since that's how we specified it in docker-compose.yml. 

So here, the command is: `docker run it <name-of-our-image>:<tags>'